GVFS/GVFS.Common/Database/PlaceholderTable.cs

```diff
@@ -157,6 +157,24 @@ public void AddPossibleTombstoneFolder(string path)
             this.Insert(new PlaceholderData() { Path = path, PathType = PlaceholderData.PlaceholderType.PossibleTombstoneFolder });
         }
 
+        public void RemoveStartingWith(string path)
+        {
+            try
+            {
+                using (IDbConnection connection = this.connectionPool.GetConnection())
+                using (IDbCommand command = connection.CreateCommand())
+                {
+                    command.CommandText = "DELETE FROM Placeholder WHERE path LIKE @path;";
+                    command.AddParameter("@path", DbType.String, $"{path}%");
+                    command.ExecuteNonQuery();
+                }
+            }
+            catch (Exception ex)
+            {
+                throw new GVFSDatabaseException($"{nameof(PlaceholderTable)}.{nameof(this.Remove)}({path}) Exception", ex);

```

Comment:
This seems like the standard pattern, but what state are things left in the exception gets thrown.  How would we suggest the user recover from that?

Reply:
Usually this would result in VFS for Git crashing and having an error entry in the logs.  If VFS for Git cannot run queries against the placeholder database then there is something really wrong and we don't want things to get worse.